Specification.md

```diff
@@ -0,0 +1,87 @@
+# XApp Specification
+
+TBD: I suppose it would be nice to call the project XApp instead of xapp ? (used lower case for now)
```

Comment:
XApp sounds good.

Reply:
Fixed: 7ddaa169f8ebfa702d07b975ad6aae174b7f973c  … @clefebvre, can you take care for the renaming of the GitHub project? (I suppose I only you have the required permissions for that)